tmt/base.py

```diff
@@ -286,20 +286,23 @@ def overview(tree):
     @staticmethod
     def create(name, template, tree, force=False):
         """ Create a new test """
+        dry = Test._opt('dry')
+
         # Create directory
         if name == '.':
             directory_path = os.getcwd()
         else:
             directory_path = os.path.join(tree.root, name.lstrip('/'))
-            tmt.utils.create_directory(directory_path, 'test directory')
+        tmt.utils.create_directory(directory_path, 'test directory', dry=dry)
```

Comment:
Why is this line moved out of the `else` clause? When creating a new test in the current directory we do not need to create it. It already exists.

Reply:
I'm sorry, I did it unintentionally and didn't notice that. I will fix!